### PR TITLE
Update dependencies to latest version to make duckling compile with ghc 8.6.3

### PR DESCRIPTION
**Pull request by mauricedoepke**
I reviewed all outdated dependencies, there are 2 or 3 with minor breaking changes but none of them affect duckling.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,17 +13,15 @@ before_cache:
 
 matrix:
   include:
-    # crashes in travis because 2 versions of unix get installed and
```
Comment (patapizza) [inline .travis.yml]: can we keep this block for further reference?

File: duckling.cabal [hunk 2]
```diff
@@ -1180,4 +1181,4 @@ executable duckling-expensive
 
 source-repository head
   type:     git
-  location: https://github.com/facebook/duckling
+  location: https://github.com/facebook/duckling
```
Comment (patapizza) [inline duckling.cabal]: nit: revert

File: .travis.yml (hunk 1 above)
Comment (mauricedoepke) [reply .travis.yml]: Sure

File: duckling.cabal (hunk 2 above)
Comment (mauricedoepke) [reply duckling.cabal]: Fixed this too

File: .travis.yml (hunk 1 above)
Comment (patapizza) [reply .travis.yml]: Thanks, seems like this line is still missing.

Comment (mauricedoepke) [reply .travis.yml]: Fixed that too.

**Comment by patapizza** [comment]
Thank you @mauricedoepke!

**Pull request closed without merging**